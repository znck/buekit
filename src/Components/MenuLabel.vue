<script>
import { style, styleResolver } from '../utils'
import { createTag } from '../mixins'

export default {
  name: 'Menu',
  functional: true,
  mixins: [createTag('p')],
  render (h, ctx) {
    const _ = styleResolver(ctx.$style)
    const { tag } = ctx.props

    return style(
      h(tag, ctx.data, ctx.children), _('menu-label')
    )
  }
}
</script>

<style lang="scss" module>
@import '~bulma/sass/utilities/_all.sass';
@import '../_base.scss';
@import '~bulma/sass/components/menu.sass';
</style>

