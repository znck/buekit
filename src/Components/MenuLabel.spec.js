import { shallow } from 'vue-test-utils'
import { createRenderer } from 'vue-server-renderer'

import MenuLabel from './MenuLabel.vue'

describe('MenuLabel.vue', () => {

  it ('matches snapshot', () => {
    const wrapper = shallow(MenuLabel, {})
    const renderer = createRenderer()

    renderer.renderToString(wrapper.vm, (err, str) => {
      if (err) throw new Error(err)
      expect(str).toMatchSnapshot()
    })
  })
})
