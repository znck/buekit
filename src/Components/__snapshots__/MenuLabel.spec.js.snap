// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuLabel.vue matches snapshot 1`] = `<p class="menu-label"></p>`;
